### DISCUSSION TITLE
Import date fields to CRM

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15720] [How to]
hi,

When i try to import the date, i am getting the error as "incorrect date value". 
![Import error](https://user-images.githubusercontent.com/96602176/154030656-3ccab45d-feec-45ac-a6c9-50b9baaa4f3f.PNG)

When i created the test record and tried to export it. The date was exported as DD-MM-YYYY format. 

![date format](https://user-images.githubusercontent.com/96602176/154030909-e7b2c383-2bc1-49ab-b230-6b231cc9e19f.PNG)

Please guide me import the records with date fields.

Thanks & regards,
Rajasekaran.

**Comment by mariuszkrzaczkowski** [comment]
What are you reporting❓  see this: https://github.com/YetiForceCompany/YetiForceCRM/discussions/15377 https://github.com/YetiForceCompany/YetiForceCRM/issues/13908
 
issues - it's not this category >> https://github.com/YetiForceCompany/YetiForceCRM/issues/new/choose
question (discussions) - then specify what you are asking

**Reply by Ilamparithi-Tamizhanangu** [reply]
Hi,

I am new to Yetiforce and posted in the discussion (How to) to clarify the doubt. 

I followed [this](https://stackoverflow.com/questions/42490101/invalid-datetime-format-1292-incorrect-datetime-value), and solved it. I just converted the date to YYYY-MM-DD format and successfully imported to Yetiforce.

Thanks & regards,
Rajasekaran.
